src/main/java/org/jboss/tools/intellij/openshift/telemetry/TelemetryService.java

```diff
@@ -34,6 +37,11 @@
     public static final String IS_OPENSHIFT = "is_openshift";
     public static final String OPENSHIFT_VERSION = "openshift_version";
 
+    public static final String DEVSANDBOX_LOGIN_DIALOG="devsandbox-login-dialog";
```

Comment:
Would be good to have every Red Hat Developer Sandbox related action starts with a common prefix and if I remember correctly _ is not be used (see @adietish)